packages/liveblocks-core/src/room.ts

```diff
@@ -67,31 +67,31 @@ type AuthCallback = (room: string) => Promise<{ token: string }>;
 
 export type Connection =
   /* The initial state, before connecting */
-  | { state: "closed" }
+  | { status: "closed" }
   /* Authentication has started, but not finished yet */
-  | { state: "authenticating" }
+  | { status: "authenticating" }
   /* Authentication succeeded, now attempting to connect to a room */
   | {
-      state: "connecting";
+      status: "connecting";
       id: number;
       userId?: string;
       userInfo?: Json;
       isReadOnly: boolean;
     }
   /* Successful room connection, on the happy path */
   | {
-      state: "open";
+      status: "open";
       id: number;
       userId?: string;
       userInfo?: Json;
       isReadOnly: boolean;
     }
   /* Connection lost unexpectedly, considered a temporary hiccup, will retry */
-  | { state: "unavailable" }
+  | { status: "unavailable" }
   /* Connection failed due to known reason (e.g. rejected). Will throw error, then immediately jump to "unavailable" state, to attempt to reconnect */
-  | { state: "failed" };
+  | { status: "failed" };
 
-export type ConnectionState = Connection["state"];
+export type ConnectionStatus = Connection["status"];
```

Comment:
While this type is renamed here, there is no impact on the public types we expose, so this isn't a breaking change. `Connection` and `ConnectionStatus` are not publicly exported types.

As you can see, these exist in the bundle as _declared_, but not _exported_ types:

![Screen Shot 2023-04-21 at 17 35 46@2x](https://user-images.githubusercontent.com/83844/233677496-2e870c59-6d82-4324-b5fd-de49624a85f5.png)

The _contents_ of `ConnectionStatus` is `"closed" | "authenticating" | ...`, so where we use those types, we not breaking any contracts. Therefore, this name change is fine.